cmd/root.go

```diff
@@ -124,6 +126,7 @@ func NewRootCommand() (*cobra.Command, error) {
 
 	cmd.Flags().StringVarP(&cfgFile, "config", "c", "", "config file (default is $HOME/prometheus-rds-exporter.yaml)")
 	cmd.Flags().BoolP("debug", "d", false, "Enable debug mode")
+	cmd.Flags().BoolP("enable-otel", "", false, "Enable OpenTelemetry")
```

Comment:
I'll maybe be more explicit about tracing if the MR is only about it because otel is traces + logs + metrics.

Reply:
Or maybe you don't need the flag at all and just use a NoopTracerProvider by default: https://pkg.go.dev/go.opentelemetry.io/otel/api/trace#NoopTracerProvider

Reply:
As discussed, I change it to `enable-otel-traces` because we may support other OTEL features (logs and metrics) in the future.